addons/mrp/static/src/mrp_forecasted/forecasted_details.xml

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="utf-8"?>
+<templates id="template">
+    <t name="mrp.ForecastedDetails" owl="1" t-inherit="stock.ForecastedDetails" t-inherit-mode="extension">
+        <xpath expr="//tr[@name='draft_picking_in']" position="after">
+            <tr t-if="props.docs.draft_production_qty.in" name="draft_mo_in">
+                <td colspan="2">Production of Draft MO</td>
+                <td t-esc="_formatFloat(props.docs.draft_production_qty.in)" class="text-end"/>
```

Comment:
nitpick: instead of `t-esc` (legacy), we should always use `t-out` now (v15 onwards) even if people still aren't consistently using it...